File: src/test/scala/cern/colt/matrix/tlong/impl/SparseLongMatrix1DTest.scala
package cern.colt.matrix.tlong.impl

import cern.colt.matrix.tlong.LongMatrix1DTest
//remove if not needed
import scala.collection.JavaConversions._

class SparseLongMatrix1DTest(arg0: String) extends LongMatrix1DTest(arg0) {

  protected def createMatrices() {
    A = new SparseLongMatrix1D(SIZE)
    B = new SparseLongMatrix1D(SIZE)
  }
}

File: src/test/java/cernx/colt/matrix/tlong/impl/SparseLongMatrix1DTest.java
package cernx.colt.matrix.tlong.impl;

import cernx.colt.matrix.tlong.LongMatrix1DTest;

public class SparseLongMatrix1DTest extends LongMatrix1DTest {

    public SparseLongMatrix1DTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseLongMatrix1D(SIZE);
        B = new SparseLongMatrix1D(SIZE);
    }
}
